uobtheatre/bookings/schema.py

```diff
@@ -35,25 +46,51 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class DiscountRequirementNode(DjangoObjectType):
+    class Meta:
+        model = DiscountRequirement
+        interfaces = (relay.Node,)
+
+
+class DiscountNode(DjangoObjectType):
+    requirements = DjangoListField(DiscountRequirementNode)
+
+    class Meta:
+        model = Discount
+        interfaces = (relay.Node,)
+
+
+class DiscountNodeMixin:
+    discounts = DjangoListField(DiscountNode)
+
+    def resolve_discounts(self, info):
+        return [
+            discount
+            for discount in self.discounts.all()
+            if not discount.is_single_discount()
+        ]
+
+
```

Comment:
I'm not sure why this is back, I think I changed this in a previous pr but I might be wrong

Reply:
Wild, I think I branched from main... resolve in the merge?